src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -313,11 +313,11 @@ const CollectorInfoComponent: React.FC = () => {
             height="100%"
             srcSet={image.srcSet}
             lazyLoad
-            alt=""
+            alt="picture of the Collection for Carole Server by Emily Johnston"
```

Comment:
This is redundant information, take a look at https://webaim.org/techniques/alttext/#context

Reply:
gonna read through that article and handle these review notes for sure